h/schemas/forms/admin/group.py

```diff
@@ -139,7 +139,7 @@ def group_organization_select_widget(_node, kwargs):
 
 class AdminGroupSchema(CSRFSchema):
     def __init__(self, *args):
-        super().__init__(validator=username_validator, *args)  # noqa: B026
+        super().__init__(*args)
```

Comment:
The `*args` looks like it's unused: we never instantiate `AdminGroupSchema()` with any arguments. But in fact Colander itself does instantiate further `AdminGroupSchema` instances for us, with arguments.